reaktive/src/jsCommonTest/kotlin/com/badoo/reaktive/scheduler/MainSchedulerTest.kt

```diff
@@ -9,4 +20,62 @@ class MainSchedulerTest {
         val executor = scheduler.newExecutor()
         executor.submit {}
     }
+
+    @Test
+    fun delayed_task_executed_with_delay(): AsyncTestResult {
+        val scheduler = MainScheduler()
+        val executor = scheduler.newExecutor()
+        var executed = false
+        executor.submit(200.milliseconds) { executed = true }
+
+        return observableOf(
+            100.milliseconds, 300.milliseconds
+        )
+            .flatMapSingle { timeout ->
+                singleTimer(timeout, scheduler = scheduler)
+                    .map { executed }
+            }
+            .toList()
+            .testAwait { (before, after) ->
+                assertFalse(before)
+                assertTrue(after)
+            }
+    }
+
+    @Test
+    fun interval_task_executed_at_interval(): AsyncTestResult {
+        val scheduler = MainScheduler()
+        val executor = scheduler.newExecutor()
+        var counter = 0L
+        val items = mutableListOf<Long>()
+
+        val delayBetweenTasks = 50.milliseconds
```

Comment:
I suggest to increase the tick delay from 50 milliseconds to 100 milliseconds, to reduce possible flakiness due to system lags.

Reply:
done